### PR TITLE
Vermont Taxable Income

### LINKED ISSUE #2399: Vermont Taxable Income
# References
Tax Form: https://tax.vermont.gov/sites/tax/files/documents/IN-111-2022.pdf
Instruction: https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf
# Example
Maine Taxable Income: https://github.com/PolicyEngine/policyengine-us/pull/2020
# Variable to add:
`vt_taxable_income` `vt_deductions` `vt_exemptions`

Vermont Taxable income equals to Vermont AGI minus deductions and exemptions
# Summary
Complete `vt_taxable_income` computation frame
Refine `vt_deductions` and `vt_exemptions` in future issues.

### PR DESCRIPTION
**Pull request by mzx-4real**
Fixes #2399

Co-authored-by: Jongwon LEE <lee.jongwon.dev@gmail.com>

### Summary
📝🧪🆕

This pull request adds Vermont taxable income as a new feature to the policy engine. It defines new variable classes for `vt_deductions`, `vt_exemptions`, and `vt_taxable_income` in the `policyengine_us/variables/gov/states/vt/tax/income/taxable_income` directory, and updates the changelog and the test suite accordingly.

> _`vt_taxable_income`_
> _New feature for Vermonters_
> _Autumn leaves fall fast_

### Walkthrough
*  Bump minor version and add Vermont taxable income feature to changelog ([link](https://github.com/PolicyEngine/policyengine-us/pull/2446/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
* Define new variable classes for Vermont income deductions, exemptions, and taxable income ([link](https://github.com/PolicyEngine/policyengine-us/pull/2446/files?diff=unified&w=0#diff-5b4da40c9563285f3becaa835d116595723e546203c2b09ecc10d719bcb77828R1-R11), [link](https://github.com/PolicyEngine/policyengine-us/pull/2446/files?diff=unified&w=0#diff-a1eac4ffeba75685bdd869990754061b5ab1aa33b98d6c511ee748b02476b197R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/2446/files?diff=unified&w=0#diff-e11f753be93d14ff247af1002726420d0bd216a347280dfab7930372cbc54facR1-R20))
* Add test case for Vermont taxable income under standard deduction scenario ([link](https://github.com/PolicyEngine/policyengine-us/pull/2446/files?diff=unified&w=0#diff-a042766e94fea1286b5f912e2b1d2834909271cab0b2913caf466069f216965aR1-R10))

### CODE REVIEW

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_exemptions.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+class vt_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont income exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.mainelegislature.org/legis/statutes/36/title36sec5126-A.html"
+    defined_for = StateCode.VT
+
+    adds = []
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_exemptions.py:13]:
```suggestion
```

Comment (PavelMakarchuk) [reply policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_exemptions.py:13]: will add once we add exemptions

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_deductions.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class vt_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont income deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_deductions.py:10]: add reference

File: policyengine_us/tests/policy/baseline/gov/states/vt/vt_taxable_income.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- name: Vermont taxable income under the standard deduction.
+  period: 2022
+  absolute_error_margin: 0
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/vt_taxable_income.yaml:3]:
```suggestion
```

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_taxable_income.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class vt_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont taxable income"
+    unit = USD
+    documentation = "VT AGI less taxable income deductions"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_taxable_income.py:9]:
```suggestion
    documentation = "VT AGI less taxable income deductions and exemptions"
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/vt_taxable_income.yaml
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/vt_taxable_income.yaml:1]: make a couple more tests for deductions & exemptions > agi & deductions & exemptions < agi

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**